lib/features/account/controller/account_controller.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:cinnamon_riverpod_2/features/account/controller/account_state.dart';
+import 'package:cinnamon_riverpod_2/infra/auth/service/auth_service.dart';
+import 'package:cinnamon_riverpod_2/infra/auth/service/firebase_auth_service.dart';
+import 'package:cinnamon_riverpod_2/infra/planner/repository/trip_repository.dart';
+import 'package:cinnamon_riverpod_2/infra/traveler/repository/traveler_repository.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final AutoDisposeNotifierProvider<AccountController, AccountState>
```

Comment:
I don't think we need need types here, usually its just like this : 
`final exampleProvider = FutureProvider<String>((ref) async => 'Hello world');
`

I know this is due to specify types rule, but is there a rule which requires to specify a return type for functions/methods?

Reply:
there is, I will replace the current one.